lib/jekyll-paginate-v2/generator/paginator.rb

```diff
@@ -20,42 +20,60 @@ def initialize(config_per_page, first_index_page_url, paginated_page_url, posts,
         @per_page = config_per_page.to_i
         @total_pages = num_pages
 
-        if @page > @total_pages
-          raise RuntimeError, "page number can't be greater than total pages: #{@page} > #{@total_pages}"
-        end
+        # RuntimeError is raised by default
+        raise "page number can't be greater than total pages: #{@page} > #{@total_pages}" if @page > @total_pages
```

Comment:
The placement of conditions at the end of lines hurts readability, it forces the dev to evaluate the entire line for the conditions. Please don't do this

Reply:
Fixed..